srcpkgs/wt/template

```diff
@@ -3,7 +3,12 @@ pkgname=wt
 version=4.10.0
 revision=2
 build_style=cmake
-makedepends="boost-devel"
+build_helper=qmake
+configure_args="-DBUILD_EXAMPLES=OFF -DINSTALL_EXAMPLES=OFF -DENABLE_QT4=OFF $(vopt_bool firebird ENABLE_FIREBIRD)"
+hostmakedepends="qt5-host-tools qt5-qmake"
+makedepends="boost-devel openssl-devel Haru-devel pango-devel sqlite-devel postgresql-libs-devel
+ libmysqlclient-devel unixodbc-devel qt5-devel libfreeglut-devel libunwind-devel
+ $(vopt_if firebird firebird3)"
```

Comment:
firebird probably needs to be added to `depends` (too?)

Reply:
I have investigated the code base a bit and it looks like it's looking for "libfbclient", which is not being provided by the firebird3 package anyway, so this option is useless